src/app/calendar/samplePage.tsx

```diff
@@ -0,0 +1,59 @@
+"use client";
+
+import React from "react";
+import Calendar, { CalendarEvent as CalendarEvent } from "../../components/Calendar/Calendar";
+import { styled } from "styled-components";
+
+const Centerer = styled.div`
+    display: flex;
+    justify-content: center;
+    align-items: center;
+`;
+const SizedAspectRatio = styled.div`
+    width: 100vmin;
+    height: 100vmin;
```

Comment:
aspect-ratio: 1 would work for most dimensions but not in cases like our wide monitor screens, where the width is much greater than the height, and so you would have to scroll to see the entire calendar (e.g. the month view). 
We take into account these cases (where the width and height differ greatly) by letting both width and height be the minimum of the two. 
 
side note: the samplePage is not a part of the component.

Reply:
ah ok this will be deleted since it's the sample page, this is fair!
it's more that the 'usual' way to do this is to fit a div to the available space and make the calendar 'fit' within it, rather than setting the height based on the viewport